ads/inabox/inabox-messaging-host.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {PositionObserver} from './position-observer';
+import {dev} from '../../src/log';
+
+/** @const */
+const TAG = 'inabox-messaging-host';
+/** @const */
+const AMP_MESSGE_PREFIX = 'amp-';
```

Comment:
I would either do "AMP_MESSAGE_PREFIX" or "AMP_MSG_PREFIX". Removing just the 'a' in 'message' is going to lead to misuse